lblod/frontend-gelinkt-notuleren
# GN-5181: add option to select chairman previous legislation

### Overview
This PR ensures that the chairman of the previous legislation may be selected as chairman and attendee in an inauguration meeting.

##### connected issues and PRs:
[GN-5181](https://binnenland.atlassian.net/browse/GN-5181?atlOrigin=eyJpIjoiMTU4MWVhMTQ0MjM4NDliMWI0MTE1OTMyMTcxZGE2MGUiLCJwIjoiaiJ9)

### Setup
Requires https://github.com/lblod/app-gelinkt-notuleren/pull/211

### How to test/reproduce
- Start the app
- Login as a gemeente/district/OCMW
- Open/create an inauguration meeting
- Ensure it's date is set so that the newly created mandatees may be selected
- Open the attendee selector modal
- Ensure that, among the mandatees, you may select the chairman of the previous legislation
- You should also be able to select that chairman as the chairman of the meeting

### Challenges/uncertainties
This PR is a bit of a hacky solution:
- It fetches the previous 'bestuursperiode'
- It then queries for a mandatee in that bestuursperiode and in the correct bestuursorgaan. That mandatee needs to have on of the following 'classificatie-codes':
   * http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000012 (voorzitter gemeenteraad)
   * http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000016 (voorzitter RvMW)
   * http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e00001c (voorzitter districtsraad)
Classificatie-code of voorzitter provincieraad is not included.

While working on the attendee modal, I also noticed it is prime for some refactoring. But due to time-constraints, I did not opt to do this refactoring in this PR:
- Addition of loading indicators
- Avoid fetching mandatees twice (once for the dropdown, and once for the table)

Additionally, I also noticed that we do not fetching the current chairman/voorzitter. We only fetching 'leden', we should probably change this.

Due to mu-cl-resources inability to filter on multiple URIs, I had to filter on the classification IDs. I added a simple `getIdentifier` util function to implement this.

### Checks PR readiness
